metriq_gym/suites/lr_qaoa.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "lr_qaoa_suite",
```

Comment:
nice that we have an LRQAOA suite!

Reply:
While testing this, I opened https://github.com/unitaryfoundation/metriq-gym/issues/498
Outside the scope of this PR, just an FYI